File: AMS.Web/src/app/models/manage-id-proof-type/id-proof-type-response.ts
import { OperationStatus } from "../shared/operation-status";
import { IdProofType } from "./id-proof-type";

export class IdProofTypeResponse 
{
    //#region Constructor

    constructor()
    {
        this.IdProofTypes = new Array<IdProofType>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    IdProofTypes: Array<IdProofType>;

    OperationStatus: OperationStatus;

    //#endregion
}

File: AMS.API/DTO/IdProofType/IdProofTypeResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.IdProofType
{
    public class IdProofTypeResponse
    {
        #region Constructor

        public IdProofTypeResponse()
        {
            IdProofTypes = new List<IdProofType>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<IdProofType> IdProofTypes { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}